doc/source/tutorials/basictutorial.rst

```diff
@@ -0,0 +1,656 @@
+at the end of the line, there are some exceptions but more about that later. ::
+
+	print "hello world".  // shows: hello world
+
+See those two forward slashes? That means that a comment has been made, anything
+after the double slashes won't be read by the script so you don't have to worry
+about what it says. Here's an example of what happens if you make something a comment: ::
+
+	// print "hello world 1".
+	print "hello world 2".
+
+If you'd run these two lines of code you'd **ONLY** see:
+``hello world 2``
+
+You'd might be wondering why you'd want to use the forward slashes if the script doesn't read them.
+The computer ignores them, but they're there to help leave explanations for the humans to see who might have to read your program.
+
+To clear the screen type: ::
+
+	clearscreen.
+
+Now say that we really like to use our hello world command but don't want to type
+the entire sentence every time, we could use the set command. ::
+
+	set x to "hello world".
+
+The set command 'sets' a certain value to the given variable. Everytime we refer
+to ``x`` we will actually refer to ``"hello world"``. ::
+
+	set x to "hello world".
+	print x. // shows: hello world
+
+Of course we can choose other things to print other than ``hello world``.
+Keep in mind that 'normal' text requires ``""`` and variables you made using the
+set command, numbers and booleans (true or false) don't need ``""``. ::
+
+	set x to "hello world".
+	set y to true.
+	set z to 123.
+
+	print x.   // shows: hello world
+	print "x". // shows: x
+	print y.   // shows: true
+	print "y". // shows: y
+	print z.   // shows: 123
+	print "z". // shows: z
+
+You can also replace a variable you've made: ::
+
+	set x to "hello world".
+	set x to "updated text".
+	print x. // shows: updated text
+
+	set x to "hello world".
+	print x. // shows: hello world
+	set x to "updated text".
+	print x. // shows: updated text
+
+As you can see ``hello world`` doesn't exist anymore. If you'd want to print
+both you could do: ::
+
+	set x to "hello world".
+	set y to x.
+	set x to "updated text".
+
+	print y. // shows: hello world
+	print x. // shows: updated text
+
+Variables don't just have to be one letter you could also use a word as a variable, don't use spaces when naming variables. ::
+
+	set WhateverThisVariableIs to false.
+	print WhateverThisVariableIs. // shows: false
+
+=============
+If statements
+=============
+
+Now we know how to set certain text to a variable we can explore more stuff.
+For instance ``if``, this checks if a certain value matches the given value. ::
+
+  set x to 1.
+
+  if x = 1 {
+    print "x is one".
+  }
+
+This will show: ``x is one``.
+
+You've probably noticed the curly brackets ``{ }`` after an ``if`` statement. You don't need a period at the end of an ``if`` statement but you need these brackets.
```

Comment:
The claim "you need these brackets" isn't really true in the example you're showing, since the example you're showing only uses one statement in the body.  It would be more accurate to say "you need these brackets *IF* you have more than one statement in the body, otherwise they're optional."  

This is valid:
```
if x = 1
  print "x is one".
```